src/cls/IPM/Utils/Update/VersionBase.cls

```diff
@@ -0,0 +1,52 @@
+Class %IPM.Utils.Update.VersionBase [ Abstract ]
```

Comment:
This should be %IPM.General.Update.VersionBase. Utils package is really just a collection of methods and no base classes as much as possible

Reply:
Also add a description to the class with intended usage and what should be subclassing it